plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/preferences/PrefPageConnectionsGeneral.java

```diff
@@ -123,12 +123,25 @@ public void widgetSelected(SelectionEvent e) {
                 GridData.FILL_HORIZONTAL,
                 300
             );
-            useWinTrustStoreCheck = UIUtils.createCheckbox(
-                settings,
-                CoreMessages.pref_page_connections_use_win_cert_label,
-                ModelPreferences.getPreferences().getBoolean(ModelPreferences.PROP_USE_WIN_TRUST_STORE_TYPE)
-            );
-            useWinTrustStoreCheck.setToolTipText(CoreMessages.pref_page_connections_use_win_cert_tip);
+            
+
+            if (System.getProperty(GeneralUtils.PROP_TRUST_STORE) != null || System.getProperty(GeneralUtils.PROP_TRUST_STORE_TYPE) != null) {
+                Composite winTrustStoreComposite = UIUtils.createComposite(settings, 1);
```

Comment:
Why do we need a separate composite just here?

Reply:
It's not possible to have a tooltip on a disabled checkbox. Disabled controls don't generate the events necessary to display a tooltip.